### ISSUE TITLE
Restore Checkpoint Error

### ISSUE THREAD

**Issue by Lyhour-Chhay** [golmschenk/Counting-with-Focus-for-Free#1]
I've been download the model "Counting with focus for free" for your gitHub. However, it still errors during the restore checkpoint when it tries to call the statement self.saver.restore(self.sess, os.path.join(checkpoint_dir, ckpt_name))  “ **Not found: Key conv1/batch_norm/BatchNorm/beta not found in checkpoint** “. Sir, do you have any suggestion or recommendations to solve this problem?

**Comment by golmschenk** [comment]
This repository is a fork. The original authors' version can be found here: https://github.com/shizenglin/Counting-with-Focus-for-Free
They would be the best people to ask.

**Comment by Lyhour-Chhay** [comment]
Thank for your replying sir. Anyway I've been notices in this code is add "Kmap" whereas the original version doesn't have. Could you tell me what is Kmap ?

**Comment by golmschenk** [comment]
I was an additional experiment my team was testing as an extension to the CFF paper. It was not completed, and is not in a working form in this repository.

**Comment by Lyhour-Chhay** [comment]
Thank you sir! :)
